test/config_parser_SUITE.erl

```diff
@@ -2587,7 +2587,10 @@ mod_shared_roster_ldap(_Config) ->
     ?errf(T(#{<<"ldap_filter">> => 1})).
 
 mod_sic(_Config) ->
-    check_iqdisc(mod_sic).
+    check_iqdisc(mod_sic),
+    T = fun(Opts) -> #{<<"modules">> => #{<<"mod_sic">> => Opts}} end,
+    M = fun(Cfg) -> modopts(mod_sic, Cfg) end,
+    ?eqf(M([]), T(#{})).
```

Comment:
This check is not doing much, but it's not harmful, we can keep it.

Reply:
Correct, I thought it won't hurt to test empty config, particularly if specification allows it.